src/frontend/style/main.css

```diff
@@ -227,8 +227,13 @@ article section p {
         position: absolute;
         right: 20px;
         top: 20px;
+        font-size:1.75em;
     }
-
+    
+	.nav-button{
+		font-size: 1.25em;
```

Comment:
This needs to get added to the block on line 76 above with the rest of the class rules for `.nav-button`